Functions/xASL_bids_BIDSifyASLJSON.m

```diff
@@ -192,23 +203,23 @@
 				warning('Did not succeed in repeating PLDs for each TE for Hadamard sequence import');
 			end
 			% Make sure that number of volumes can be divided by the repeated PLDs
-			jsonOut.PostLabelingDelay = repeatedPLDs;
+			jsonOut.PostLabelingDelay = repeatedPLDs; %do we want to repeat the PLDs??
 		end
 	end
 end
 
 % Check for FME Hadamard sequences
 if isfield(jsonOut,'SeriesDescription')
-	isHadamardFME = ~isempty(regexp(char(jsonOut.SeriesDescription),'(Encoded_Images_Had)\d\d(_)\d\d(_TIs_)\d\d(_TEs)', 'once'));
-	if isHadamardFME
+	bHadamardFME = ~isempty(regexp(char(jsonOut.SeriesDescription),'(Encoded_Images_Had)\d\d(_)\d\d(_TIs_)\d\d(_TEs)', 'once'));
+	if bHadamardFME
 		startDetails = regexp(char(jsonOut.SeriesDescription),'\d\d(_)\d\d(_TIs_)\d\d(_TEs)', 'once');
 		jsonOut.HadamardType = xASL_str2num(jsonOut.SeriesDescription(startDetails:startDetails+1),'auto');
```

Comment:
The option 'auto' here doesn't exist, what does this mean? Same below

Reply:
It did.... That's really weird. @jan-petr & I added that option a while ago when we had issues with the writing of JSON files. I can't find it right now though. I'm checking the history. Is someone overwriting code?!? We even use it in scripts like `spm_jsonwrite`.

Reply:
https://github.com/ExploreASL/ExploreASL/pull/414/files

Reply:
I think we've / I've been mixing this up 😞 We should check the code for that.